src/main/java/seedu/contax/logic/parser/BatchCommandParser.java

```diff
@@ -14,6 +14,9 @@
 
 
 public class BatchCommandParser implements Parser<BatchCommand> {
+
+    public static String MESSAGE_PREFIX_NOT_EQUALS_ONE = "Only one value of param (=, start or end) can be provided.";
```

Comment:
I suggest a rephrase here.
```suggestion
    public static String MESSAGE_PREFIX_NOT_EQUALS_ONE = "Only one parameter (=, start or end) can be provided.";
```

Reply:
thx changed